flake8_rst/rst.py

```diff
@@ -3,13 +3,14 @@
 
 RST_RE = re.compile(
     r'(?P<before>'
-    r'^(?P<indent> *)\.\. (code-block|sourcecode):: (python|pycon)\n'
+    r'^(?P<indent> *)\.\. (code-block|sourcecode|ipython):: (python|pycon)\n'
```

Comment:
If i read properly, ipython [directive](https://ipython.org/ipython-doc/rel-0.13.2/development/ipython_directive.html) use without (python|pycon)

And ipython console output contain square brackets, without `>>>`, how does it work?

Reply:
maybe I've been a little bit too enthusiastic, however in [pandas](https://github.com/pandas-dev/pandas/blob/master/doc/source/enhancingperf.rst) they use directives like
`.. ipython:: python` followed by plain python code. [with some roles like :suppress: sometimes]

These cases would be covered as well.

Reply:
okey